syntax/ts-common/members.vim

```diff
@@ -34,6 +35,8 @@ syntax keyword typescriptAccessibilityModifier public private protected containe
 
 syntax keyword typescriptReadonlyModifier readonly override contained
 
+syntax keyword typescriptAutoAccessor accessor
```

Comment:
@HerringtonDarkholme Ah, I'm sorry that I missed `contained` at end of this line